contrib-prompt/functions/prompt_sorin-contrib_setup

```diff
@@ -0,0 +1,132 @@
+# An extension of the existing sorin theme to add some contrib features.
+#
+# Authors:
+#   Jonathan Pigrée <jonathan.pigree@gmail.com>
+#
+# Screenshots:
+#   http://i.imgur.com/nrGV6pg.png
+#
+
+#
+# 16 Terminal Colors
+# -- ---------------
+#  0 black
+#  1 red
+#  2 green
+#  3 yellow
+#  4 blue
+#  5 magenta
+#  6 cyan
+#  7 white
+#  8 bright black
+#  9 bright red
+# 10 bright green
+# 11 bright yellow
+# 12 bright blue
+# 13 bright magenta
+# 14 bright cyan
+# 15 bright white
+#
+
+# Load dependencies.
+pmodload 'helper'
+
+function load_sorin {
```

Comment:
This shouldn't be needed. Loading contrib-prompt then prompt should put sorin in the fpath and it should be set up for autoloading. If that doesn't work, there's probably an issue I need to fix.

Reply:
You are right. It works without this code. Awesome.